src/components/activity-page/managed-interactive/iframe-runtime.tsx

```diff
@@ -58,6 +59,9 @@ export const IframeRuntime: React.FC<IProps> =
       phone.addListener("hint", (newHint: any) => {
         setNewHint(newHint.text || "");
       });
+      phone.addListener("showModal", (modalProps: any) => {
+        toggleModal(modalProps);
```

Comment:
Is the `modelProps` eventually going to be needed by the `toggleModal` function?  I see that the `toggleModal` function passed via props doesn't use it, but I'm assuming it's going to be needed at some point.